src/DDTrace/ScopeManager.php

```diff
@@ -33,18 +33,17 @@ public function getActive()
             return null;
         }
 
-        return $this->scopes[count($this->scopes) - 1];
+        return end($this->scopes);
```

Comment:
Apparently, we are interested in returning the last element, so maybe `end()` is better?

Reply:
end() modifies the array's internal pointer, which may or may not surprise us somewhere else. I don't think the original line is unclear.